internal/cachecontroller/cache_controller.go

```diff
@@ -47,6 +46,8 @@ var (
 	}, []string{"invalidation_required", "changes_size"})
 )
 
+const OneYear = time.Hour * 24 * 365
```

Comment:
Perhaps this belongs in `pkg/storage/cache.go` as MAX_TIME so we don't get bitten once again by the internals of the cache implementation.